source/reference/minio-server/settings/kes.rst

```diff
@@ -51,27 +80,54 @@ Refer to your operating system's documentation for how to define an environment
 
    See the :minio-git:`KES wiki <kes/wiki/Configuration#policy-configuration>` for more complete documentation on KES access control.
 
-.. envvar:: MINIO_KMS_KES_KEY_NAME
+   You must also set the :envvar:`MINIO_KMS_KES_KEY_FILE`.
+   This variable is mutually exclusive with :envvar:`MINIO_KMS_KES_API_KEY`.
 
-   The name of an external key on the Key Management system (KMS) configured on the KES server and used for performing en/decryption operations. 
-   MinIO uses this key for the following:
+.. envvar:: MINIO_KMS_KES_CAPATH
+   :optional:
 
-   - Encrypting backend data (:ref:`IAM <minio-authentication-and-identity-management>`, server configuration).
+   Allows validation of the KES Server Certificate for a Self-Signed or Third-Party :abbr:`CA <Certificate Authority>`.
+   Specify the path to the location of the :abbr:`CA <Certificate Authority>` certificate for your KES deployment.
```

Comment:
```suggestion
   Allows validation of the KES Server Certificate for a Self-Signed or Third-Party :abbr:`CA (Certificate Authority)`.
   Specify the path to the location of the :abbr:`CA (Certificate Authority)` certificate for your KES deployment.
```

Reply:
Right, because we couldn't use the same marking punctuation. 😂 
Thanks for the catch and fix! I forever will get these wrong.